app/controllers/rents_controller.rb

```diff
@@ -0,0 +1,13 @@
+class RentsController < ApplicationController
+  include Wor::Paginate
```

Comment:
What do you think about move `include Wor::Paginate` to ApplicationController?

Reply:
That's great, this way i wont have to write that line in every ModelController!